morpheus/app/views.py

```diff
@@ -116,15 +117,17 @@ class EmailSendView(ServiceView):
         return PreResponse(text='201 job enqueued\n', status=201)
 
 
-async def check_sms_limit(conn, company_code):
+async def get_sms_spend(conn, company_code, start, end):
     v = await conn.fetchval(
         """
-        select sum(m.cost)
-        from messages as m
-        join message_groups j on m.group_id = j.id
-        where j.company=$1 and send_ts > (current_timestamp - '28days'::interval)
+        select sum(cost)
+        from messages
+        join message_groups j on group_id = j.id
+        where j.company=$1 and send_ts between $2 and $3
```

Comment:
most of the time you don't actually care about finish, and this will be called a lot. Make `end` optional.

use buildpg, but we will need to add `between` as a function there.

Reply:
`and cost is not null`